public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Question.js

```diff
@@ -1,9 +1,13 @@
 import React, { PropTypes } from 'react';
 import {ManagedForm, ManagedField} from '../../../ManagedEditor';
 import FormFieldTextInput from '../../../FormFields/FormFieldTextInput';
-import FormFieldArrayWrapper from '../../../FormFields/FormFieldArrayWrapper';
-import {StoryQuestionsAnswer} from './Answer';
+import SearchSuggestions from '../../../FormFields/SearchFields/SearchSuggestions';
 import uuidv4 from 'uuid/v4';
+import { fetchCapiAtom } from '../../../../services/capi';
+
+const filters = {
```

Comment:
I'd maybe abstract this out into the `consts` folder, not that it's any more robust, but it makes it easier to reuse if needed.